azure-pipelines.yml

```diff
@@ -0,0 +1,69 @@
+# nimi-bot pipeline for running all system tests as part of Continuous Integration.
+
+parameters:
+- name: module_names
+  type: object
+  default: # Order of the modules is based on test execution time, longest firt.
+  - nidigital
+  - nitclk  
+  - nifgen
+  - nidcpower
+  - nidmm
+  - niscope
+  - nimodinst
+  - nise
+  - niswitch
+
+# A pipeline with no CI trigger
+trigger: none
+
+# specific path
```

Comment:
I don't think this comment is accurate, not sure where it came from. Should be removed or improved `# When should the nimi-bot CI be kicked off.`

Reply:
This pipeline will trigger only for the PR runs not after submitting the changes.

Reply:
yeah but why does the comment say "specific path" ?

Reply:
I have removed the comment.

Reply:
Generally we provide include paths in this case we are only providing exclude paths. So I removed the comment.